package.json

```diff
@@ -119,6 +119,6 @@
     "restify": "^6.3.4",
     "rimraf": "^2.6.2",
     "tmp": "0.0.33",
-    "typescript": "~2.6.2"
+    "typescript": "~2.8.1"
```

Comment:
I think @ofrobots was wary of upgrading typescript to 2.8. Ali?

Reply:
Seems fine to me. The issue would be if we started exporting types that depended on users having typescript 2.8+ to be able to parse our `.d.ts` files. I don't think that's the case here.